play_by_play.py

```diff
@@ -125,6 +91,8 @@ def _fix_data_types(self, plays_dataframe):
         plays_dataframe['PLAYER1_TEAM_ID'] = plays_dataframe['PLAYER1_TEAM_ID'].astype('Int64')
```

Comment:
These constants should be saved in a file somewhere. 

For example you can create something like this

```
class ColumnKeys:
    PLAYER1_TEAM_ID = 'PLAYER1_TEAM_ID'
```
This reduces the chance of you having String typo errors.